crates/typed-store/Cargo.toml

```diff
@@ -22,7 +22,7 @@ hdrhistogram = "7.5.1"
 rocksdb = { version = "0.19.0", features = ["snappy", "lz4", "zstd", "zlib", "multi-threaded-cf"], default-features = false }
 serde = { version = "1.0.140", features = ["derive"] }
 thiserror = "1.0.37"
-tokio = { workspace = true, features = ["full", "test-util"] }
+tokio = { version = "1", features = ["full", "test-util"] }
```

Comment:
`full` includes the components of `test-util`

Reply:
no it doesn't. I had to add that manually to make it compile.

Reply:
I mean: https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml#L88

Reply:
I don't think so? https://github.com/tokio-rs/tokio/blob/master/tokio/Cargo.toml#L29

Reply:
Not following what you mean?

Reply:
`full` contains `rt, sync, time`, which are the only contents of `test-util`

Reply:
unless `test-util` is required directly, that is :detective:

Reply:
I think @ljedrz was saying the `test-util` feature is included in the `full` features but it doesn't look like it is which would explain why you needed both there.

Reply:
It is needed for some macros that test code uses.

Reply:
Because it depends on conditionally compiled code depending on that feature.

Reply:
yeah, makes sense; I just never saw it used explicitly in the tokio APIs and thought it was just a bundle of features used for test purposes :+1: